flake.nix

```diff
@@ -7,7 +7,7 @@
       flake = false;
     };
     gomod2nix = {
-      url = "github:tweag/gomod2nix";
+      url = "github:yihuang/gomod2nix";
```

Comment:
https://github.com/tweag/gomod2nix/pull/69
upstream PR not merged yet, but it could be slow merged.